src/Symfony/Component/Validator/Resources/translations/validators.ro.xlf

```diff
@@ -398,6 +398,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Valoarea netmask-ului trebuie sa fie intre {{ min }} si {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Denumirea fișierului este prea lungă. Ea trebuie să conțină {{ filename_max_length }} caractere sau mai puține.|Denumirea fișierului este prea lungă. Ea trebuie să conțină {{ filename_max_length }} caractere sau mai puține.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Complexitatea parolei este prea mică. Vă rugăm să folosiți o parolă mai puternică.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Această valoare conține caractere care nu sunt premise de nivelul de restricționare curent.</target>
```

Comment:
`nivelul curent de restricție`?

Reply:
Thank you for the feedback @nicolas-grekas. I still believe `restricționare` makes more sense in the context, since `restricție` means more like a rule and `restricționare` means the process of enforcing that rule. Either place of the word `curent` sounds correct to me, so I wasn't sure which one to choose. Can we reach a middle ground by having `nivelul curent de restricționare` ?

Reply:
you're the native speaker :)